geoips/filenames/base_paths.py

```diff
@@ -80,11 +83,15 @@ def initialize_paths():
 
     # Output Directories
     if not os.getenv("GEOIPS_OUTDIRS"):
-        raise KeyError(
-            "GEOIPS_OUTDIRS must be set in your environment. "
-            "Please set GEOIPS_OUTDIRS and try again"
+        LOG.warning(
+            "GEOIPS_OUTDIRS is not set in your environment. "
+            "Defaulting to current directory as output dir. "
+            "Please set GEOIPS_OUTDIRS if you want to write to a different directory "
+            "by default."
         )
-    paths["GEOIPS_OUTDIRS"] = os.getenv("GEOIPS_OUTDIRS").rstrip("/")
+        paths["GEOIPS_OUTDIRS"] = os.getcwd()
```

Comment:
I would recommend using $HOME/geoips_outdirs, just so we always end up in the same place.

Reply:
Changed! I used all caps for GEOIPS_OUTDIRS.... should it be lowercase?

Reply:
It is fine!